### DISCUSSION TITLE
Valeurs possibles pour le champ status de payment order

### DISCUSSION THREAD

**Discussion by bcohen-io** [PayGreen/documentation#2] [Q&A] [answered]
Bonjour,

Quelles valeurs peut-on recevoir pour le champ status reçu lorsqu'on fait un https://developers.paygreen.fr/reference/get_get_payment_order ?
Sur la doc tout indique que c'est un objet (voir capture ci-jointe), mais en réalisant 2/3 tests on s'aperçoit que c'est une chaine du genre payment_order.expired
![image](https://user-images.githubusercontent.com/104826013/232062583-13d63c49-c9f1-4578-bcdd-8f100732747a.png)

Merci

**Comment by AdrienBr** [comment, marked as answer]
Bonjour @bcohen-io , 
Vous trouverez la liste des statuts sur ce guide : https://developers.paygreen.fr/docs/payment-payment-orders#status

Bonne journée,
